geth/jail/jail_cell.go

```diff
@@ -1,38 +1,31 @@
 package jail
 
 import (
-	"time"
+	"sync"
 
 	"fknsrs.biz/p/ottoext/fetch"
 	"fknsrs.biz/p/ottoext/loop"
 	"fknsrs.biz/p/ottoext/timers"
-	"github.com/eapache/go-resiliency/semaphore"
 	"github.com/robertkrimen/otto"
-
-	"github.com/status-im/status-go/geth/common"
 )
 
 const (
-	// JailCellRequestTimeout seconds before jailed request times out.
-	JailCellRequestTimeout = 60
+	// JCellRequestTimeout seconds before jailed request times out.
+	JCellRequestTimeout = 60
 )
 
-// JailCell represents single jail cell, which is basically a JavaScript VM.
-type JailCell struct {
-	// FIXME(tiabc): It's never called. Double-check and delete.
-	//sync.Mutex
+// JCell represents single jail cell, which is basically a JavaScript VM.
+type JCell struct {
```

Comment:
Please, refrain from refactoring in regular tasks: https://hackmd.io/s/rkgqzRWIb

You can add a `TODO` if you want to rename it under some refactoring later.